src/components/App/app.scss

```diff
@@ -0,0 +1,19 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&display=swap');
```

Comment:
Are you sure you need to import all those font sizes? This adds a lot of weight to your app.

Reply:
Besides, is there a special reason to import fonts here and not use a ‘link‘ element in your html header?

Reply:
Nope, `<link>` is enough